attributes/default.rb

```diff
@@ -395,11 +395,13 @@
 )
 # Lustre repo string is built following the official doc
 # https://docs.aws.amazon.com/fsx/latest/LustreGuide/install-lustre-client.html
+# 'centos' is used for arm and 'el' for x86_64
+default['cfncluster']['lustre']['base_url_prefix'] = arm_instance? ? 'centos' : 'el'
```

Comment:
Minor: Can we put `centos` or `centos7` in the variable name so we know this prefix is only used by Centos7? We should also point this out in the new comment above

Reply:
ok done